code/game/world.dm

```diff
@@ -485,7 +495,9 @@ GLOBAL_VAR(restart_counter)
 			CRASH("Unsupported platform: [system_type]")
 
 	var/init_result = call_ext(library, "init")("block")
-	if (init_result != "0")
+	if(length(init_result) != 0 && init_result[1] == ".") // if first character is ., then it returned the output filename
```

Comment:
Returned path will *always* start with a `.`, this isn't platform or setup-specific behavior or anything.

See: https://github.com/ParadiseSS13/byond-tracy/blob/d59c014d3685e3491284e129f80e930934944285/prof.c#L1477